src/modules/uORB/uORBDeviceMaster.hpp

```diff
@@ -33,7 +33,8 @@
 
 #pragma once
 
-#include <stdint.h>
+#include <vector>
```

Comment:
vector is not available on NuttX

Reply:
Yes it is.

Reply:
Yup, my bad. But the usage of STL is problematic on NuttX (and similar on Posix, but we don't expect std::bad_alloc to be thrown there):
- the STL inherently uses and depends on exceptions. In particular std::bad_alloc. Bad things happen if they're disabled. Firefox for example disables exceptions as well, and they do not use the STL for that reason (they have their own framework, MFBT)
- the STL might pull in symbols from libstdc++ (and I'm happy to be proven wrong here). This can depend on the toolchain and its version. Meaning that it might compile for us but not for someone else, or in a future toolchain version.